ig/input/pagecontent/internals.md

```diff
@@ -1,3 +1,16 @@
 ### Overview
 
-This is where we talk about GitHub and our internal interfaces.
+The ReportStream Intermediary is microservice behind the larger ReportStream system.  This page documents some of the internals behind the microservice.
```

Comment:
Is an `a` missing?: `is microservice` => `is **a** microservice`

Reply:
Good catch.  English hard.

Reply:
Fixed.